subsys/bluetooth/controller/Kconfig

```diff
@@ -7,6 +7,56 @@
 
 comment "BLE Controller support"
 
+# The following symbols are enabled depending if the controller actually
+# supports the respective features. In case of Nordic SW LL, all of these are
+# supported; for OpenISA SW LL, they're not
+
+config BT_LL_SW_LEGACY_SUPPORT
+	bool
+	depends on SOC_COMPATIBLE_NRF
+
```

Comment:
Can we reorder the features to match the ordering in Table 4.5 in Core v5.1, Vol 6, Part B, Section 4.6?

Reply:
Sure. I've reordered them, and I will re-push. Do let me know if it looks OK.